src/jsonb_utils.c

```diff
@@ -239,3 +239,47 @@ ts_jsonb_get_interval_field(const Jsonb *json, const char *key)
 
 	return DatumGetIntervalP(interval_datum);
 }
+
+Jsonb *
+ts_errdata_to_jsonb(ErrorData *edata)
```

Comment:
Did u implemented it from scratch or copied from Postgres source code? If copied is good to place a comment about it and from where you copied

Reply:
I wrote it from scratch